Data/MultiSet.hs

```diff
@@ -207,8 +211,42 @@ instance Ord a => Semigroup (MultiSet a) where
       | otherwise = MS $ Map.map (* fromIntegral n) m
 #endif
 
+-- | Note that 'elem' is slower than 'member'.
 instance Foldable.Foldable MultiSet where
-    foldr = fold
+    foldr = foldr
```

Comment:
> Change Foldable.foldr to use top level foldr function.

The `foldr` on the right already means the top-level `foldr` from this module.

```suggestion
    foldr = fold
```

Would this be less confusing?